$MirOS: ports/misc/screen/patches/shmpatch-configure_in,v 1.1.7.1 2005/03/18 15:50:35 tg Exp $
$OpenBSD: shmpatch-configure_in,v 1.1 2003/11/22 00:56:58 espie Exp $
--- configure.in.orig	2003-06-03 11:58:02.000000000 +0000
+++ configure.in	2004-06-11 15:38:21.000000000 +0000
@@ -1203,18 +1214,6 @@ main() {
   exit(0); /* libc version works properly.  */
 }], AC_DEFINE(USEMEMCPY))
 
-AC_MSG_CHECKING(long file names)
-(echo 1 > /tmp/conftest9012345) 2>/dev/null
-(echo 2 > /tmp/conftest9012346) 2>/dev/null
-val=`cat /tmp/conftest9012345 2>/dev/null`
-if test -f /tmp/conftest9012345 && test "$val" = 1; then
-AC_MSG_RESULT(yes)
-else
-AC_MSG_RESULT(no)
-AC_DEFINE(NAME_MAX, 14)
-fi
-rm -f /tmp/conftest*
-
 AC_MSG_CHECKING(for vsprintf)
 AC_TRY_LINK(,[vsprintf(0,0,0);], AC_MSG_RESULT(yes);AC_DEFINE(USEVARARGS), AC_MSG_RESULT(no))
 
